### PR TITLE
Feature/np translations

### PR DESCRIPTION
**Pull request by Vearso**
Changes from another pull request made and added translations files

### CODE REVIEW

File: src/components/Cart/Steps/Steps.vue [hunk 1]
```diff
@@ -0,0 +1,67 @@
+<template>
+  <div class="np-steps">
+    <div v-for="(stepName, index) in $tm('menuSteps',{returnObjects: true})"
+         :key="index"
+         class="np-steps__wrapper">
+
+      <div :class="{
```
Comment (entrop-) [inline src/components/Cart/Steps/Steps.vue:7]: if theres more complicated logic then simple comparison, then move it to methods (setup) and just call function here
somsething like calculateClass(index)

File: src/components/Cart/Summary/Summary.vue [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+.np-cart__summary {
+  @apply w-full px-12 py-6;
+
+  &__price {
+    @apply flex justify-between;
+  }
+
+  &__button {
+    @apply w-full bg-primary font-bold mt-4 p-1;
+    color: #fff;
```
Comment (entrop-) [inline src/components/Cart/Summary/Summary.vue:58]: i think theres text-white in tailwind

File: src/components/Header/Header.vue [hunk 3]
```diff
@@ -1,13 +1,50 @@
...
+<script lang="ts">
 export default {
-name: "Header"
+  props: {
+    image: {
+      type: String,
+      default: ""
+    },
+  },
+  setup(){
```
Comment (entrop-) [inline src/components/Header/Header.vue:22]: just remove setup if its empty

File: src/components/Cart/Summary/Summary.vue [hunk 4]
```diff
@@ -0,0 +1,61 @@
+<template>
+  <div class="np-cart__summary">
+    <div class="np-cart__summary__price">
+      <p>{{ $t(totalPrice) }}}</p>
```
Comment (KamilWozniak) [inline src/components/Cart/Summary/Summary.vue:4]: masz trzy nawiasy zamykające

**Review by KamilWozniak — APPROVED** [review]
aproved

**Pull request merged**